data/items.js

```diff
@@ -4035,6 +4035,66 @@ exports.BattleItems = {
 		gen: 3,
 		desc: "Cannot be eaten by the holder. No effect when eaten with Bug Bite or Pluck.",
 	},
+	"poweranklet": {
+		id: "poweranklet",
+		name: "Power Anklet",
+		isUnreleased: false,
```

Comment:
Someone just today came into the help chat asking to use these items because they needed it for their trick room team. If we're looking to provide an accurate simulation for the cartridge games, shouldn't all items be available? In standard formats I would agree, but with VGC having Item Clause, these items being on Showdown is pretty necessary, I would say.